ee/dataflatten/jwt.go

```diff
@@ -31,9 +24,20 @@ func flattenJWT(path string, opts ...FlattenOpts) ([]Row, error) {
 		return nil, fmt.Errorf("unable to read JWT: %w", err)
 	}
 
+	return flattenJWT(tokenRaw, opts...)
+}
+
+func JWT(rawdata []byte, opts ...FlattenOpts) ([]Row, error) {
```

Comment:
Added this so that `dataflattentable.DataSourceType` `JWTType` was fully implemented with both a file and a data function -- it is unused.